style.css

```diff
@@ -1,3 +1,220 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
-  background-color: blueviolet;
+  background-color: #eee;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.container {
+  width: 80%;
```

Comment:
- [ ] Why 80? The page should be full width.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/92411708-5cb70580-f11f-11ea-99d0-ab42f3ac2aa4.png) |![image](https://user-images.githubusercontent.com/61479648/92411760-8c660d80-f11f-11ea-8495-13b789af36d8.png)  |